ui_framework/components/tabs/_tabs.scss

```diff
@@ -23,6 +27,11 @@
   border-radius: 0; /* 1 */
   margin-bottom: -1px; /* 3 */
 
+  @include darkTheme {
+    background-color: $tabBackgroundColor--darkTheme;
+    border-color: $tabBorderColor--darkTheme;
```

Comment:
I think we'll also need to add a `color` property here to make it match the link's dark theme style (https://github.com/elastic/kibana/blob/master/ui_framework/components/_mixins.scss#L25).

```scss
color: $globalLinkColor--darkTheme;
```

Similar changes will need to be made to the hover and focus states.